components/scream/src/physics/common/physics_constants.hpp

```diff
@@ -64,6 +64,7 @@ struct Constants
   static constexpr Scalar P0            = 100000.0;        // reference pressure, Pa
   static constexpr Scalar RD            = 287.15;          // gas constant for dry air, J/kg/K
   static constexpr Scalar RHOSUR        = P0/(RD*Tmelt);
+  static constexpr Scalar rhosui        = 60000/(RD*253.15);
```

Comment:
Do we have (or do we want) a naming convention for constants? It's not your job to regularize them in this PR--I'm just curious.

Reply:
@jeff-cohere feel free to choose a convention and sed-script it in a PR. My advice is to follow the Fortran casing where it looks like the Fortran one was cased thoughtfully.